disperser/common/v2/blob.go

```diff
@@ -71,7 +71,7 @@ func BlobStatusFromProtobuf(s pb.BlobStatus) (BlobStatus, error) {
 
 // BlobMetadata is an internal representation of a blob's metadata.
 type BlobMetadata struct {
-	BlobHeader *core.BlobHeader
+	BlobHeader *corev2.BlobHeader
```

Comment:
ah, this is not just a doc issue, it's a real bug. Our test seems weak.

Reply:
Not sure. The go compiler does the right thing with `core "github.com/Layr-Labs/eigenda/core/v2"`